### PR TITLE
Params in db

### PR DESCRIPTION
**Pull request by IsaacHaze**
initial work in progess stuff: storing the model parameters in the sqlite backend

### CODE REVIEW

File: semanticizest/tests/test_semanticizer.py [hunk 1]
```diff
@@ -35,3 +39,31 @@ def test_semanticizer_redirect():
...
+            with open(join(dirs['actual'], fname), 'w') as out:
+                tokens = f.read().split()
+                out.write("\n".join(str(cand) for cand in sem.all_candidates(tokens)))
+        with open(join(dirs['expected'], fname)) as f:
+            expected = f.read()
+        with open(join(dirs['actual'], fname)) as f:
+            actual = f.read()
+
+        # assert_multi_line_equal(expected, actual)
+    return 1
```
Comment (larsmans) [inline semanticizest/tests/test_semanticizer.py]: This assert should be on, right?

File: semanticizest/_semanticizer.py [hunk 2]
```diff
@@ -9,29 +10,24 @@
...
-        create the stored model.
-    """
+    def __init__(self, fname):
+        """Create a semanticizer from a stored model.
+
+        Parameters
+        ----------
+        fname : string
+            Filename of the stored model from which to load the
+            Wikipedia statistics.
```
Comment (larsmans) [inline semanticizest/_semanticizer.py]: Docstring on class, not `__init__` (for easy reference in IPython).

File: semanticizest/_semanticizer.py [hunk 3]
```diff
@@ -43,7 +39,20 @@ def __init__(self, fname, N=7):
...
         self.commonness = commonness
-        self.N = N
+        self.N = self._get_ngram_max_length()
+
+    def _get_ngram_max_length(self):
+        self._cur.execute("select value "
+                          "from parameters "
+                          "where key = 'N';")
+        N = int(self._cur.fetchone()[0])
+        return N
```
Comment (larsmans) [inline semanticizest/_semanticizer.py]: Nitpick: no need to bind to `N` just before return.

File: semanticizest/tests/test_semanticizer.py (hunk 1 above)
Comment (IsaacHaze) [inline semanticizest/tests/test_semanticizer.py]: True, fixed in 4b7544c

**Pull request merged**